storm-core/src/clj/org/apache/storm/ui/core.clj

```diff
@@ -863,6 +855,7 @@
                                       (.get_eventlog_host comp-page-info)
                                       (.get_eventlog_port comp-page-info)
                                       secure?)
+       "profilingAndDebuggingCapable" (not on-windows?)
```

Comment:
Can we file a follow on JIRA to make this a bit cleaner?  Perhaps a script for different environments that we can check at run time if the script is there?